pub_sub/python/sdk/order-processor/app.py

```diff
@@ -26,4 +26,4 @@ def orders_subscriber():
         'ContentType': 'application/json'}
 
 
-app.run(port=5001)
+app.run(port=6001)
```

Comment:
Could we please use the APP_PORT env variable instead like other quickstarts, versus hard code?  then your Dapr Run will drive it.

Reply:
Example:

https://github.com/dapr/quickstarts/blob/master/bindings/python/sdk/batch/app.py#L24